### DISCUSSION TITLE
I build it using Arduino IDE and CLI - interested?

### DISCUSSION THREAD

**Discussion by ubidefeo** [simpleavr/s14clock#6] [General]
If anyone is interested in building this in Arduino IDE or using the CLI I'm happy to provide a PR.

The changes are pretty straight forward, took me one minute.
It will, of course, retain compatibility with PlatformIO as per @simpleavr 's preference

u.

**Comment by simpleavr** [comment]
Thanks for @ubidefeo's contribution, the Arduino setup (and other fixes) is now available, see README.md.
